documentation/slate/source/includes/timetables.md

```diff
@@ -1,11 +1,29 @@
-<a name="timetables"></a>Timetables
+<a name="timetables"></a>![Timetables](/images/line_schedules.png)Timetables
 =================================
 
+The Timetables features gives you access to line schedules on public transport, 
+allowing you to find the times public transport is expected at specific stops.
+
+It returns a variety of differents informations, including: stop schedules, origin and destination, 
+timetables, and accessibility information.
+
+In order to display line schedules, you may have to use these APIs (click on them for details):
+
+-   **[Public transportation objects](#pt-ref)** : List of the public transport
+    objects of a region
+
+| url | Result |
+|---------------------------------------------------------|-------------------------------------|
```

Comment:
same here, not convinced that you need to put a link to /coverage and it you do I think it's better to put it below